modules.d/95fcoe/lldpad.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if ! getargbool 0 rd.nofcoe ; then
+if getargbool 0 rd.nofcoe ; then
```

Comment:
Sorry for being late, but I think it's very confusing:
- **nofcoe** means _don't enable fcoe_
- 0 for getargbool means _false_
hence
**nofcoe=0** means _don't enable fcoe == false_ which is equivalent to _enable fcoe == true_
which is exactly the opposite of what we want here.

Reply:
@danimo please ignore, answered in https://github.com/dracutdevs/dracut/commit/6f7823bce65dd4b52497dbb94892b637fd06471a